Doc added for ASI system configuration (#177)
magento/merchdocs: src/configuration/advanced/system.md

|Maximum Width|Global|Determines the maximum pixel width for the image. When the image is resized, it does not exceed this width. Default: 1920|
|Maximum Height|Global|Determines the maximum pixel height for the image. When the image is resized, it does not exceed this height. Default: 1200|

## Adobe Stock Integration

![]({% link images/images/system-adobe-stock-integration.png %}){: .zoom}
_Adobe Stock Integration_

|Field|[Scope]({% link configuration/scope.md %})|Description|
|--- |--- |--- |
|Enabled Adobe Stock|Global|Switch that enables or disables the Adobe Stock Integration|

> |API Key (Client ID)|Global|An API key is required to connect your Magento application to the Adobe Stock service. You can get an API key by creating an integration at [console.adobe.io](https://console.adobe.io/).|

Reviewer: Let's add a placeholder link to the future overview article here:

```suggestion
|API Key (Client ID)|Global|An API key is required to connect your Magento application to the Adobe Stock service. You can get an API key by [following our documentation on the topic][3].|
```

Reviewer 2: I will add link after merge

> |Client Secret|Global|A Client Secret is required. You can get a Client Secret by creating an integration at [console.adobe.io](https://console.adobe.io/)|

Reviewer: Let's add a placeholder link to the future overview article here:

```suggestion
|Client Secret|Global|A Client Secret is required. You can get a Client Secret by [following our documentation on the topic][3]|
```

Reviewer 2: I will add link after merge

|Test Connection||Run a test checking that the API key is valid for use with Adobe Stock's service.|

[1]: https://en.wikipedia.org/wiki/Sendmail

> [2]: https://magento.com/security/news/new-zend-framework-1-security-vulnerability

Reviewer: Let's add a third reference here, like:

```
[3]: {% link catalog/adobe-stock.md %}
```

This will link up to the article I am working on in #82

Author: I think merchdocs are using these references for third-party URLs which are not in merchdocs so I suggest to use `[following our documentation on the topic]({% link catalog/adobe-stock.md %})` instead of mentioning in reference.

Reviewer 2: Yes, we use them for links outside of the site build. SO also for items in our devdocs or other Magento sites, but not topics that are in the same project.

Reviewer 2: I will add link after merge

|Maximum Width|Global|Determines the maximum pixel width for the image. When the image is resized, it does not exceed this width. Default: 1920|
|Maximum Height|Global|Determines the maximum pixel height for the image. When the image is resized, it does not exceed this height. Default: 1200|

<!--{% if "Default.CE Only" contains site.edition %}-->
## Adobe Stock Integration

![]({% link images/images/system-adobe-stock-integration.png %}){: .zoom}
_Adobe Stock Integration_

|Field|[Scope]({% link configuration/scope.md %})|Description|
|--- |--- |--- |

> |Enabled Adobe Stock|Global|Enable Adobe Stock Integration|

Reviewer: ```suggestion
|Enabled Adobe Stock|Global|Switch that enables or disables the Adobe Stock Integration|
```

> |API Key (Client ID)|Global|API key is required to connect Magento application to Adobe Stock. You can get API key after registering an Adobe Stock API key at [console.adobe.io](https://console.adobe.io/).|

Reviewer: ```suggestion
|API Key (Client ID)|Global|An API key is required to connect your Magento application to the Adobe Stock service. You can get an API key by creating an integration at [console.adobe.io](https://console.adobe.io/).|
```

> |Client Secret|Global|You can get Client Secret after registering an Adobe Stock API key at [console.adobe.io](https://console.adobe.io/)|

Reviewer: ```suggestion
|Client Secret|Global|A Client Secret is required. You can get a Client Secret by creating an integration at [console.adobe.io](https://console.adobe.io/)|
```

> |Test Connection||Validates the API key for Adobe Stock Integration connection.|

Reviewer: ```suggestion
|Test Connection||Run a test checking that the API key is valid for use with Adobe Stock's service.|
```
